### DISCUSSION TITLE
update messed up my setup

### DISCUSSION THREAD

**Discussion by Zarphus** [Raytwo/ARCropolis#597] [Issues] [answered]
can we get a release of the 13.0.2 build without the updater? i need to maintain this version for the moment, to ensure stability for an upcoming event.

**Comment by Raytwo** [comment]
It's already been up since yesterday...

**Reply by Zarphus** [reply, marked as answer]
the update does not run on 13.0.2, when launching, it says it requires 13.0.3 to run. 
however, we decided to just get all our devices updated for the event anyway so for me in particular, its a non-issue now.

**Reply by Raytwo** [reply]
My bad, I misread what you were asking for, but the auto-updater is a setting IIRC so you could’ve disabled that
